### PR TITLE
Replaced tabs with four spaces

### PR DESCRIPTION
**Pull request by erikrikarddaniel**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint .`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

I updated `*.nf` files in `subworkflows/local`, `workflows` and `modules/local` and `*.config` files so tabs at the start of lines were replaced with four spaces.

### CODE REVIEW

File: modules/local/dada2_taxonomy.nf [hunk 1]
```diff
@@ -67,7 +67,7 @@ process DADA2_TAXONOMY {
         Genus = tx\$tax.Genus,
         confidence = tx\$confidence,
         sequence = tx\$sequence,
- 	row.names = names(seq)
+     row.names = names(seq)
```
Comment (d4straub) [inline modules/local/dada2_taxonomy.nf:70]: I dont get it. In the visualization here it looks worse, [in the current dev branch it looks better](https://github.com/nf-core/ampliseq/blob/dev/modules/local/dada2_taxonomy.nf) than [in the PR here](https://github.com/erikrikarddaniel/ampliseq/blob/fix-tabs/modules/local/dada2_taxonomy.nf).

File: modules/local/dada2_taxonomy.nf [hunk 2]
```diff
@@ -67,7 +67,7 @@ process DADA2_TAXONOMY {
...
    )
```
Comment (d4straub) [inline modules/local/dada2_taxonomy.nf:71]: This bracket seems to have a weird indentation in the current dev and here.

**Comment by d4straub** [comment]
For `modules/local/dada2_taxonomy.nf` it seems worse to me and one "wrong" indentation was not corrected.
For  `workflows/ampliseq.nf` it looks better though.

File: modules/local/dada2_taxonomy.nf (hunk 1 above)
Comment (erikrikarddaniel) [reply modules/local/dada2_taxonomy.nf]: You're right. I must have edited this with my standard 2 space indentation sometime ago, and that got weird when I now updated the file automatically. I fixed this particular line in 21637e2. Did you see any other mistakes?

**Comment by d4straub** [comment]
Here seems something wrong as well: https://github.com/nf-core/ampliseq/blob/dev/modules/local/cutadapt_summary.nf
I think I made that file. Maybe thats an unrelated problem?

File: conf/modules.config [hunk 3]
```diff
@@ -83,9 +83,9 @@ params {
             publish_dir = './'
             publish_files = null
         }
-	'itsx_cutasv' {
+    'itsx_cutasv' {
             args = '-t all --preserve T --date F --positions F --graphical F --save_regions none'
-	}
+    }
```
Comment (d4straub) [inline conf/modules.config:88]: this didnt work out

File: modules/local/dada2_addspecies.nf [hunk 4]
```diff
@@ -53,8 +53,8 @@ process DADA2_ADDSPECIES {
         Genus = tx[,"Genus"],
         Species = tx[,"Species"],
         confidence = tx[,"confidence"],
-	sequence = tmp[,],
-	row.names=row.names(tmp)
+    sequence = tmp[,],
+    row.names=row.names(tmp)
```
Comment (d4straub) [inline modules/local/dada2_addspecies.nf:57]: also strange

File: modules/local/format_taxresults.nf [hunk 5]
```diff
@@ -23,8 +23,8 @@ process FORMAT_TAXRESULTS {
         path(fastafile)
 
         output:
-	path("ASV_tax.tsv")
-	path("ASV_tax_species.tsv"), emit: tsv
+    path("ASV_tax.tsv")
+    path("ASV_tax_species.tsv"), emit: tsv
```
Comment (d4straub) [inline modules/local/format_taxresults.nf:27]: not good

**Review by d4straub — COMMENTED** [review]
Rest looks fine to me.

File: modules/local/format_taxresults.nf (hunk 5 above)
Comment (erikrikarddaniel) [reply modules/local/format_taxresults.nf:27]: The above three should be fixed in c2930bbb7218e17e079750dea6c8040fd4a440d8.

**Comment by erikrikarddaniel** [comment]
2 space indentation converted to 4 space in 12c8efcd007e186584dd9ac4650c37aa9e841a20.

**Review by d4straub — APPROVED** [review]
Thanks! This was more work than anticipated!

**Pull request merged**